test/aircraft/Pilot/Pilot.spec.js

```diff
@@ -619,14 +638,97 @@ ava('.conductInstrumentApproach() returns a success message', (t) => {
     t.deepEqual(result, expectedResult);
 });
 
+ava('.crossFix() returns early when the specified fix does not exist', (t) => {
+    const aircraftModel = new AircraftModel(ARRIVAL_AIRCRAFT_INIT_PROPS_MOCK);
+    const invalidFixNameMock = 'threeve';
+    const altitudeMock = 13000;
+    const expectedResult = [false, 'unable to find \'threeve\''];
+    const result = aircraftModel.pilot.crossFix(aircraftModel, invalidFixNameMock, altitudeMock);
+
+    t.deepEqual(result, expectedResult);
+});
+
+ava('.crossFix() returns early when the specified fix exists but is not on the aircraft\'s route', (t) => {
+    const aircraftModel = new AircraftModel(ARRIVAL_AIRCRAFT_INIT_PROPS_MOCK);
+    const nonRouteFixNameMock = 'ogkij';
+    const altitudeMock = 13000;
+    const expectedResult = [false, 'unable, \'ogkij\' is not on our route'];
+    const result = aircraftModel.pilot.crossFix(aircraftModel, nonRouteFixNameMock, altitudeMock);
+
+    t.deepEqual(result, expectedResult);
+});
+
+ava('.crossFix() returns early when the specified altitude fails .validateNextAltitude()', (t) => {
+    const aircraftModel = new AircraftModel(ARRIVAL_AIRCRAFT_INIT_PROPS_MOCK);
+    const fixNameMock = 'kepec';
+    const altitudeMock = 13000;
+    const expectedResult = [false, 'some reply from validate-next-altitude'];
+
+    sinon.stub(aircraftModel, 'validateNextAltitude', () => expectedResult);
+
+    const result = aircraftModel.pilot.crossFix(aircraftModel, fixNameMock, altitudeMock);
+
+    t.deepEqual(result, expectedResult);
+});
+
+ava('.crossFix() correctly configures arrival aircraft\'s MCP and returns correct response when provided valid parameters', (t) => {
+    const aircraftModel = new AircraftModel(ARRIVAL_AIRCRAFT_INIT_PROPS_MOCK);
+    const fixNameMock = 'kepec';
+    const altitudeMock = 13000;
+    const expectedResult = [true, {
+        log: 'cross KEPEC at 13000',
+        say: 'cross kepec at one three thousand'
+    }];
+    const result = aircraftModel.pilot.crossFix(aircraftModel, fixNameMock, altitudeMock);
+
+    t.deepEqual(result, expectedResult);
+});
+
+ava('.crossFix() correctly configures departure aircraft\'s MCP and returns correct response when provided valid parameters', (t) => {
+    const aircraftModel = new AircraftModel(DEPARTURE_AIRCRAFT_INIT_PROPS_MOCK);
+    const fixNameMock = 'cowby';
+    const altitudeMock = 13000;
+    const expectedResult = [true, {
+        log: 'cross COWBY at 13000',
+        say: 'cross cowby at one three thousand'
+    }];
+    const result = aircraftModel.pilot.crossFix(aircraftModel, fixNameMock, altitudeMock);
+
+    t.deepEqual(result, expectedResult);
+});
+
+// ava('.crossFix() correctly configures MCP and returns correct response ', (t) => {
```

Comment:
do we plan to refactor this test in the future or is this no longer needed?  if the former, lets un-comment it but list it as skip: `ava.skip()`.  If the later, lets remove it.

Reply:
If we need that test we should write it...instead of skipping it 😉

Reply:
nope, it's left over copy pasta 😃 all gone now.